_haikus/bunny.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: haiku
+title: bunny is fat
+author: sciduck
+---
+my bunny is fat
```

Comment:
Add either two spaces `  ` or a `<br>` tag at the end of each line to force a line break in markdown.